Resources/Locale/en-US/accent/skeleton.ftl

```diff
@@ -0,0 +1,32 @@
+accent-skeleton-words-1 = fuck you
+accent-skeleton-words-replace-1 = I've got a BONE to pick with you
+
+accent-skeleton-words-2 = fucked
+accent-skeleton-words-replace-2 = boned
+
+accent-skeleton-words-3 = fuck
+accent-skeleton-words-3-2 = fck
+accent-skeleton-words-3-3 = shit
+accent-skeleton-words-replace-3 = babushka
```

Comment:
wtf?

Reply:
ahhyes i was copying "preset" from russian and then replaced words